Example/Vouched/FaceViewController.swift

```diff
@@ -15,6 +15,7 @@ class FaceViewController: UIViewController {
     @IBOutlet private weak var instructionLabel: UILabel!
 
     private var helper: VouchedCameraHelper?
+    var isLivenessJob: Bool = false
```

Comment:
In this file, we are reusing the existing take a selfie view controller, and adding a conditional that will call the new selfie verification foo.